src/main.rs

```diff
@@ -24,7 +30,7 @@ fn main() {
                 .long("database")
                 .value_name("FILE")
                 .help("Path to the Mentat database to serve")
-                .default_value("temp.db")
+                .default_value("")
```

Comment:
That was an earlier review comment of mine. Using `""` here means "default to an in-memory database", which I think is sensible.